tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/processor/Processor.scala

```diff
@@ -54,62 +55,65 @@ trait Processor extends Serializable {
     futures.clear()
   }
 
-  def extraValue(row: Row, field: String, toBytes: Boolean = false): Any = {
+  def extraValue(row: Row,
+                 field: String,
+                 fieldTypeMap: Map[String, DataType],
+                 toBytes: Boolean = false): Any = {
     // TODO
     val index = row.schema.fieldIndex(field)
-    row.schema.fields(index).dataType match {
+    fieldTypeMap(field) match {
       case StringType =>
         val result = if (!row.isNullAt(index)) {
-          row.getString(index).mkString("\"", "", "\"")
+          row.getAs[String](index).mkString("\"", "", "\"")
         } else {
           "\"\""
         }
         if (toBytes) result.getBytes else result
       case ShortType =>
         if (!row.isNullAt(index)) {
-          row.getShort(index)
+          row.getAs[Short](index)
         } else {
           0.toShort
         }
       case IntegerType =>
         if (!row.isNullAt(index)) {
-          row.getInt(index)
+          row.getAs[Int](index)
         } else {
           0
         }
       case LongType =>
         if (!row.isNullAt(index)) {
-          row.getLong(index)
+          row.getAs[Long](index)
         } else {
           0L
         }
       case FloatType =>
         if (!row.isNullAt(index)) {
-          row.getFloat(index)
+          row.getAs[Float](index)
```

Comment:
what's different ？

Reply:
change it back and support more datatype conversion

Reply:
I think use `getAs` maybe not a good idea otherwise why support another method